core/server/common/src/main/java/alluxio/master/journal/raft/SnapshotReplicationManager.java

```diff
@@ -254,9 +265,6 @@ public long maybeCopySnapshotFromFollower() {
     if (mDownloadState.get() == DownloadState.DOWNLOADED) {
       return installDownloadedSnapshot();
     }
-    if (mDownloadState.get() == DownloadState.REQUEST_DATA) {
```

Comment:
why removing this check?
how will we guard against a specific follower becoming idle going forward?

Reply:
In `requestData()` we now have
```
RaftClientReply reply = mJournalSystem.sendMessageAsync(peerId,
                toMessage(JournalQueryRequest.newBuilder()
                    .setSnapshotRequest(GetSnapshotRequest.getDefaultInstance()).build()))
            .get(SNAPSHOT_REQUEST_TIMEOUT_MS, TimeUnit.MILLISECONDS);
```
here these requests time out automatically.